src/Sylius/Bundle/AdminBundle/Resources/config/services/twig.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ This file is part of the Sylius package.
+  ~
+  ~ (c) Sylius Sp. z o.o.
+  ~
+  ~ For the full copyright and license information, please view the LICENSE
+  ~ file that was distributed with this source code.
+  -->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="Twig\Extension\StringLoaderExtension">
```

Comment:
Does sylius have any policies, that 3rd party classes should have string ID instead of FQCN?

For reason, that sylius don't control release cycles and BC promise of those packages and DI definition may conflict with some theoretical twig extension bundle.

Like
```suggestion
        <service id="sylius.***.string_loader" class="Twig\Extension\StringLoaderExtension">
```